backend/corpora/common/entities/project.py

```diff
@@ -1,9 +1,65 @@
 from .entity import Entity
-from ..corpora_orm import DbProject
+from ..corpora_orm import DbProject, DbProjectLink
 
 
 class Project(Entity):
     table = DbProject
 
     def __init__(self, db_object: DbProject):
         super().__init__(db_object)
+
+    @classmethod
+    def create(
+        cls,
+        status: str,
+        name: str = "",
+        description: str = "",
+        owner: str = "",
+        s3_bucket: str = "",
+        tc_uri: str = "",
+        needs_attestation: bool = False,
+        processing_state: str = "",
+        validation_state: str = "",
+        links: list = None,
+    ) -> "Project":
+        """
+        Need to check if one exists before creating
+        :param id:
```

Comment:
Delete lines 27-38 as they don't provide additional context to the docstring.

Reply:
done